byzcoin/service.go

```diff
@@ -1015,31 +1089,27 @@ func (s *Service) createStateChanges(coll *collection.Collection, scID skipchain
 	// we need to find out if this is as expensive as it looks, and if so if
 	// we could use some kind of copy-on-write technique.
 
-	cdbTemp := coll.Clone()
```

Comment:
The comment above here can go away, since the new data structure was designed for a lightweight clone.